cli/cmd/install.go

```diff
@@ -54,7 +54,7 @@ If you are sure you'd like to have a fresh install, remove these resources with:
 Otherwise, you can use the --ignore-cluster flag to overwrite the existing global resources.
 `
 
-	errMsgLinkerdConfigResourceConflict = "Can't install the Linkerd control plane in the '%s' namespace. Reason: %s.\nIf this is expected, use the --ignore-cluster flag to continue the installation.\n"
+	errMsgLinkerdConfigResourceConflict = "Can't install the Linkerd control plane in the '%s' namespace. Reason: %s.\nRun `linkerd upgrade` with the relevant binary, if you are looking to upgrade Linkerd.\n"
```

Comment:
Do we need to make a reference to the "relevant binary"? Could  we say something like "run the command `linkerd upgrade` ?

Reply:
I agree. Just saying to run the `upgrade` command should be good.